### PR TITLE
Updates package.manifest example with missing info about bundleOptions

### PR DESCRIPTION
**Pull request by warrenbuckley**
We had missing info about a property available in our package.manifest
https://github.com/umbraco/Umbraco-CMS/pull/10691

### CODE REVIEW

File: Extending/Property-Editors/package-manifest.md [hunk 1]
```diff
@@ -228,6 +233,13 @@ The parameter editors array follows the same format as the property editors desc
 }
 ```
 
+## Bundling
+`bundleOptions` is an enum that expects one of the following values:
+
+* `Default` - The default bundling behavior for assets in the package folder where the assets will be bundled with the typical packages bundle.
+* `None` - The assets in the package will not be processed at all and will all be requested as individual assets and will essentially be a bundle that has composite processing turned off for both debug and production
+* `Independent` - The packages assets will be processed as it's own separate bundle. (in debug, files will not be processed)
```
Comment (busrasengul) [inline Extending/Property-Editors/package-manifest.md:241]:
```suggestion
* `Independent` - The packages assets will be processed as its own separate bundle. (In debug, files will not be processed)
```

**Review by busrasengul — COMMENTED** [review]
Thank you @warrenbuckley 
Added tiny tweaks. If you can apply them it's good to merge!

File: Extending/Property-Editors/package-manifest.md [hunk 2]
```diff
@@ -228,6 +233,13 @@ The parameter editors array follows the same format as the property editors desc
 }
 ```
 
+## Bundling
+`bundleOptions` is an enum that expects one of the following values:
```
Comment (sofietoft) [inline Extending/Property-Editors/package-manifest.md:237]: I don't know the suggested changes here makes sense, but `enum` threw me off a bit 🤔 
Perhaps we could loose the ´type´?

```suggestion
`bundleOptions` is an enumerable type that expects one of the following values:
```

**Review by sofietoft — COMMENTED** [review]
Great additions 💪 

I've added a single suggestion - let me know if it makes sense!

**Comment by warrenbuckley** [comment]
@sofietoft & @busrasengul

**Review by sofietoft — APPROVED** [review]

**Comment by sofietoft** [comment]
Looks great @warrenbuckley !

Merging in 💪

**Pull request merged**